.github/actions/pr-review/pr_review.py

```
@@ -1007,6 +1018,81 @@ def model_binding(mode: str) -> str:
+    """
+    entries = []
+    for finding in findings[:MAX_LEDGER_ENTRIES]:
+        entries.append(
+            {
+                "f": finding_fingerprint(finding),
+                "p": finding.path[:200],
+                "l": finding.line if isinstance(finding.line, int) else None,
+                "s": finding.severity,
+                "t": " ".join(finding.title.split())[:MAX_LEDGER_TITLE],
+            }
+        )
```

Ruff (0.16.1)

[warning] Use a list comprehension to create a transformed list

(PERF401)